website/blog/2025-02-19-react-native-0.78.md

```diff
@@ -0,0 +1,217 @@
+
+With this release, we simplified the process to enable the React Compiler in your React Native apps. [In previous versions](https://react.dev/learn/react-compiler#using-react-compiler-with-react-17-or-18), you had to install two packages: the compiler and its runtime. After those packages were installed, you had to configure a Babel plugin to enable React Compiler through Metro.
+
+Now, you only need to install the compiler itself and to configure the Babel plugin. To learn how to enable it, you can follow our step-by-step [guide](https://react.dev/learn/react-compiler#usage-with-babel).
+
+To verify that the compiler is running, you can open the React Native DevTools: you should see that the components that are memoized have the `Memo ✨` tag attached to them in the Component Inspector.
+
+If you want to learn more about React Compiler, these are useful resources:
+
+- [React Compiler](https://react.dev/learn/react-compiler) docs
+- [React Compiler Deep Dive](https://www.youtube.com/watch?v=uA_PVyZP7AI)
+
+### Towards smaller and faster releases
+
+We’re updating our release process to ship stable React Native releases more frequently in 2025.
+
+It will be easier for you to update the React Native version because we’ll be reducing the number of breaking changes we ship. Faster releases also means that all the bugfixes we ship internally are reaching you earlier, and you can benefit from the latest features we develop inside React Native.
+
+We believe this new model will benefit every developer in the React Native ecosystem, as fewer breaking changes means a more stable framework that everyone can rely on.
+
+### Opt-in for JavaScript logs in Metro
+
+We've added an opt-in to restore JavaScript log streaming via the Metro dev server, [previously removed in 0.77](/blog/2025/01/21/version-0.77#removal-of-consolelog-streaming-in-metro) for Community CLI users. This is in response to user feedback, as well as reviewing where we are with our replacement offerings today.
+
+To opt in, use the new `--client-logs` flag. This can also be aliased via an npm script for convenience.
+
+```sh
+npx @react-native-community/cli start --client-logs
+```
+
+Log streaming in Metro will still be going away in future and remains off by default. However, we intend to give developers a longer migration period to adapt to this change.
+
+This update will also be made available in the incoming 0.77.1 minor release.
+
+### Better Android SVG support through Vector Drawable
```

Comment:
Another nit: The docs on `VectorDrawable` describes only generic vector graphics, and not necessarily the SVG format. Could be confusing since `react-native-svg` is not a part of React Native core.

Reply:
The section title could be rephrased to something like "Added support for Android XML drawables", because this is net-new and not limited to just vector drawables. We support loading every drawable type (shapes, layer lists, etc.) so I don't want to imply that it's only vector drawables

Reply:
> Another nit: The docs on VectorDrawable describes only generic vector graphics, and not necessarily the SVG format. Could be confusing since react-native-svg is not a part of React Native core.

@huntie We have a warning box at the end to avoid confusion with `react-native-svg`! 😉 

---

@Abbondanzo I remember some back and forth with this title on the doc. I think we ended up with this title because it was unclear what XML drawables are. I'm happy to revert, if that's more accurate. I'll talk with @cortinico tomorrow.

Reply:
Let's go for `Added support for Android XML drawables` as this captures also other non-vector drawable resources